pkg/pillar/hypervisor/kvm.go

```diff
@@ -103,6 +103,16 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   chardev = "charmonitor"
   mode = "control"
 
+[chardev "charlistener"]
+  backend = "socket"
+  path = "` + kvmStateDir + `{{.DisplayName}}/listener"
```

Comment:
instead of a listener, lets at least call it listener.qmp

Reply:
Done